runtime/handler.go

```diff
@@ -11,6 +12,11 @@ import (
 	"golang.org/x/net/context"
 )
 
+var (
+	// The indent to use for JSON responses.
+	ResponseIndent = ""
```

Comment:
Generally I see public variables in golang as constants, but there are exceptions - I would generally do something like:

```
var (
  responseIndent = ""
)

func SetResponseIndent(s string) {
  responseIndent = s
}
```

But just my opinion.